recipe/meta.yaml

```diff
@@ -1,6 +1,9 @@
 {% set name = "xgboost" %}
 {% set version = "1.7.6" %}
-{% set build_number = 5 %}
+{% set build_number = 6 %}
+
+{% set string_prefix = "cuda" ~ (cuda_compiler_version | replace('.', '')) %}  # [cuda_compiler_version != "None"]
+{% set string_prefix = "cpu" %}  # [cuda_compiler_version == "None"]
```

Comment:
Think we can drop this in our case as we don't use it

For context in the conda-forge feedstock this is used to distinguish CPU/CUDA packages. However we fold both of those packages together in RAPIDS and always build CUDA-enabled packages. So the distinction isn't relevant in RAPIDS

```suggestion
```